ecommerce_api_client/auth.py

```diff
@@ -1,24 +1,40 @@
+# pylint: disable=duplicate-code
+
 import jwt
+import datetime
+
 from requests.auth import AuthBase
 
 
 class JwtAuth(AuthBase):
     """Attaches JWT Authentication to the given Request object."""
 
-    def __init__(self, username, full_name, email, signing_key, tracking_context=None):
+    def __init__(self, username, full_name, email, signing_key, tracking_context=None, issuer=None, expires_in=30):
+        self.issuer = issuer
+        self.expires_in = expires_in
         self.username = username
         self.email = email
         self.full_name = full_name
         self.signing_key = signing_key
         self.tracking_context = tracking_context
 
+    def datetime_now(self):
+        return datetime.datetime.utcnow()
```

Comment:
@jimabramson This produces the following error: `can't set attributes of built-in/extension type 'datetime.datetime'`